site/en/docs/extensions/mv3/match_patterns/index.md

```diff
@@ -3,159 +3,76 @@ layout: 'layouts/doc-post.njk'
-         <td>Matches any local file whose path starts with <code>/foo</code></td>
-         <td>file:///foo/bar.html<br>file:///foo</td>
-      </tr>
-      <tr>
-         <td><code>http://127.0.0.1/*</code></td>
-         <td>Matches any URL that uses the <code>http</code> scheme and is on the host 127.0.0.1</td>
-         <td>http://127.0.0.1/<br>http://127.0.0.1/foo/bar.html</td>
-      </tr>
-      <tr>
-         <td><code>http://localhost/*</code></td>
-         <td>Matches any localhost port</td>
-         <td>http://localhost:3000<br>http://localhost:8080</td>
-      </tr>
-      <tr>
-         <td><code>*://mail.google.com/*</code></td>
-         <td>Matches any URL that starts with <code>http://mail.google.com</code> or <code>https://mail.google.com</code>.</td>
-         <td>http://mail.google.com/foo/baz/bar<br>https://mail.google.com/foobar</td>
-      </tr>
-      <tr>
-         <td><code>urn:*</code></td>
-         <td>Matches any URL that starts with <code>urn:</code>.</td>
-         <td>urn:uuid:54723bea-c94e-480e-80c8-a69846c3f582<br>urn:uuid:cfa40aff-07df-45b2-9f95-e023bcf4a6da</td>
-      </tr>
-   </tbody>
-</table>
-
-### ❌ Invalid patterns {: #invalid-patterns }
-
-Here are some examples of _invalid_ match patterns:
-
-<table class="fixed-table width-full">
-   <tbody>
-      <tr>
-         <th style="margin-left:0; padding-left:0">Invalid pattern</th>
-         <th style="margin-left:0; padding-left:0">Why it's invalid</th>
-      </tr>
-      <tr>
-         <td><code>https://www.google.com</code></td>
-         <td>No <em>path</em></td>
-      </tr>
-      <tr>
-         <td><code>https://*foo/bar</code></td>
-         <td>'*' in the <em>host</em> can be followed only by a '.' or '/'</td>
-      </tr>
-      <tr>
-         <td><code>https://foo.*.bar/baz&nbsp;</code></td>
-         <td>If '*' is in the <em>host</em>, it must be the first character</td>
-      </tr>
-      <tr>
-         <td><code>http:/bar</code></td>
-         <td>Missing <em>scheme</em> separator ("/" should be "//")</td>
-      </tr>
-      <tr>
-         <td><code>foo://*</code></td>
-         <td>Invalid <em>scheme</em></td>
-      </tr>
-      <tr>
-         <td><code>chrome://*</code></td>
-         <td>Unsupported <em>scheme</em></td>
-      </tr>
-      <tr>
-         <td><code>chrome-extension://*</code></td>
-         <td>Unsupported <em>scheme</em></td>
-      </tr>
-      <tr>
-         <td><code>about:*</code></td>
-         <td>Unsupported <em>scheme</em>.</td>
-      </tr>
-   </tbody>
-</table>
-
-{% Aside 'success' %}
-Content scripts can be injected into related frames like `about:`, `data:`, etc by setting `"match_origin_as_fallback"` to `"true"` in the manifest. See [Injecting in related frames][cs-frames] for details.
-{% endAside %}
+: Chrome doesn't support match patterns for [top Level domains (TLD)][mdn-tld]. Specify your match patterns within individual TLDs, as in `http://google.es/*` and `http://google.fr/*`.
```

Comment:
What does this mean? That you can't have `*.co.uk`?

Reply:
The original says you can't do `google.*`. `*.example.com` is allowed.

Reply:
It would need to be `*://*.co.uk`, which is [in one of our unit tests](https://source.chromium.org/chromium/chromium/src/+/main:extensions/common/url_pattern_unittest.cc;l=1056;drc=707b3abc38ded8ab7eb0fa21664b1556324929b4). I'm told that gets the same result as `<all_urls>`. So, should we care?

Reply:
@oliverdunk

Reply:
The TLDs section was ported directly from the extension's FAQs article. This may have been so in an older version of Chrome, since @oliverdunk has been able to test that content scripts can be injected into TLDs match patterns.

Reply:
@AmySteam, I didn't follow that.

Reply:
@oliverdunk told me over chat that the current text is fine.l